src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderLiteratureRequest.js

```diff
@@ -0,0 +1,94 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+import { withCancel } from '@api/utils';
+import { documentRequestApi } from '@api/documentRequests/documentRequest';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@routes/urls';
+import { Message } from 'semantic-ui-react';
+import { Loader } from '@components/Loader';
+import _ from 'lodash';
+
+export class OrderLiteratureRequest extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cancellableLiterature: {},
+      isLoading: true,
+      hasError: false,
+    };
+  }
+
+  componentDidMount() {
+    const {
+      orderId: { pid },
+    } = this.props;
+    this.fetchLiteratureRequest(pid);
+  }
+
+  componentWillUnmount() {
+    this.cancellableLiteratureRequest &&
+      this.cancellableLiteratureRequest.cancel();
+  }
+
+  fetchLiteratureRequest = async (pid) => {
+    try {
+      this.cancellableLiteratureRequest = withCancel(
+        documentRequestApi.list(
+          documentRequestApi.query().withPhysicalItemProviderPid(pid).qs()
+        )
+      );
+      const response = await this.cancellableLiteratureRequest.promise;
+      this.setState({
+        cancellableLiterature: response.data,
```

Comment:
I think the naming is a bit misleading for this state, since it should keep document requests not literature, and I am not sure why `cancellableLiterature`